core/core/initutils.go

```diff
@@ -273,10 +273,15 @@ func getAttackTracksGetter(ctx context.Context, attackTracks, accountID string,
 
 // getUIPrinter returns a printer that will be used to print to the program’s UI (terminal)
 func getUIPrinter(ctx context.Context, verboseMode bool, formatVersion string, attackTree bool, viewType cautils.ViewTypes) printer.IPrinter {
-	p := printerv2.NewPrettyPrinter(verboseMode, formatVersion, attackTree, viewType)
+	var p printer.IPrinter
+	if helpers.ToLevel(logger.L().GetLevel()) >= helpers.WarningLevel {
```

Comment:
@vladklokun Now we will print the summary table only when the logger level is Info or Debug.